dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/NetUtils.java

```diff
@@ -206,4 +215,8 @@ public static boolean ignoreNetworkInterface(NetworkInterface networkInterface)
                 || !networkInterface.isUp();
     }
 
+    public static boolean isSpecifyNetworkInterface(NetworkInterface networkInterface) {
+        String preferredNetworkInterface = System.getProperty(DOLPHIN_SCHEDULER_PREFERRED_NETWORK_INTERFACE);
+        return Objects.equals(networkInterface.getDisplayName(), preferredNetworkInterface);
+    }
```

Comment:
Hi,

Good job,
`Utils` should be used as a tool class and it is better not to involve business process.
Is it better to implement this part in the business process class?

Reply:
- Hello, this is exclusive to net, other business is not involved, so I think it is better to achieve here,how do you think？

Reply:
> * Hello, this is exclusive to net, other business is not involved, so I think it is better to achieve here,how do you think？

Hi,

I think it is better not to add the `isSpecifyNetworkInterface` in `findNetworkInterface` of `utils`, but I find there is only one call of `findNetworkInterface`, so I don't think of a better place to add this at present.
I think we can think about the implement scheme again later.

Reply:
> I think this modify like this, the name is more better.
> 
> ```
>  private static boolean isSpecifyNetworkInterface(NetworkInterface networkInterface) {
>         String preferredNetworkInterface = PropertyUtils.getString(DOLPHIN_SCHEDULER_PREFERRED_NETWORK_INTERFACE);
>         logger.info("config name:{}, net work interface: {}, addrs: {}",
>                 preferredNetworkInterface,  networkInterface, networkInterface.getInterfaceAddresses());
>         return Objects.equals(networkInterface.getName(), preferredNetworkInterface) || 
>                 Objects.equals(networkInterface.getDisplayName(), preferredNetworkInterface);
>     }
> ```
> 
> common.properties add interface config
> 
> ```
> # Network Interface name
> #dolphin.scheduler.network.interface.preferred=eth0
> ```